lib/PHPExif/Mapper/ImageMagick.php

```diff
@@ -228,7 +226,7 @@ protected function extractGPSCoordinates(string $coordinates) : float|false
         if (is_numeric($coordinates) === true) {
             return ((float) $coordinates);
         } else {
-            $m = '!^([1-9][0-9]*\/[1-9][0-9]*), ([1-9][0-9]*\/[1-9][0-9]*), ([1-9][0-9]*\/[1-9][0-9]*)!';
+            $m = '!^([0-9]+\/[1-9][0-9]*), ([0-9]+\/[1-9][0-9]*), ([0-9]+\/[1-9][0-9]*)!';
```

Comment:
The regex looks good. But I had a look at the method `normalizeComponent` below and spotted another inconsistency.

If the regular expression does not match, we return `false` to indicate an error or absent of a valid value or error.

If the regular expression matches, but the components cannot be normalized into a `float`, e.g. in case of a string like `0/0` which would result into an division-by-zero error, we then return `0.0` as the default value. I wonder if we shouldn't return `false` in these cases, too.

Reply:
I'm inclined to agree. I was trying to find examples of problem values that led to that, but I can't. #14 seems to be where that was added, but I can't find a related issue in Lychee.